src/main/resources/default/templates/biz/tenants/user-accounts-list.html.pasta

```diff
@@ -47,6 +47,13 @@
                                                     url="@apply('/user-accounts/select/%s', account.getIdAsString())"/>
                                 </t:dropdownSection>
                             </i:if>
+                            <i:if test="UserContext.get().getUser().hasPermission('flag-system-support')">
```

Comment:
The permission `flag-system-support` is an OXOMI thing. But can't we just use the `permission-select-tenant` instead? Question is valid for all places you evaluate this flag.